### PR TITLE
Add support for fallbacks

### PR DESCRIPTION
**Pull request by jordanoverbye**
I've added a new option `fallback` to the `app-static` package so it can support client-side routing. I've also updated the docs for this package.

### CODE REVIEW

File: packages/app-static/README.md [hunk 1]
```diff
@@ -5,6 +5,42 @@ title: KeystoneJS Static File App
...
+
+The path to serve files from.
+
+### `src`
+
+The path to the folder containing static files.
+
+#### `fallback` (optional)
+
+The name of the file to serve if none is found.
```
Comment (mitchellhamilton) [inline packages/app-static/README.md]:
```suggestion
The path to the file to serve if none is found. This path is resolved relative to the `src` path.
```

Comment (jordanoverbye) [reply packages/app-static/README.md]: Thanks @mitchellhamilton !

File: packages/app-static/README.md [hunk 2]
```diff
@@ -5,6 +5,42 @@ title: KeystoneJS Static File App
...
+const { StaticApp } = require('@keystone-alpha/app-static');
+
+module.exports = {
+  new Keystone(),
+  apps: [
+    new GraphQLApp(),
+    new AdminUIApp(),
+    new StaticApp({
+      path: '/',
+      src: pathModule.join(__dirname, 'build'),
```
Comment (MadeByMike) [inline packages/app-static/README.md]: Remove `pathModule.join(...)` and replace with just `src`. I think this is a simpler usage example.

**Comment by MadeByMike** [comment]
@all-contributors please add @jordanoverbye for code

**Pull request merged**